src/encoded/static/components/motifs.js

```diff
@@ -175,16 +190,63 @@ export class MotifElement extends React.Component {
         const targetListLabel = (targetList.indexOf(',') !== -1) ? 'Targets' : 'Target';
         const pwmsLabel = (pwmsLength > 1 || pwmsLength === 0) ? 'PWMs' : 'PWM';
         const footprintsLabel = (footprintsLength > 1 || footprintsLength === 0) ? 'Footprints' : 'Footprint';
+        
+        const targetComponent = (
+            <React.Fragment>
+            {targetList.length > 0 && (
+                <React.Fragment>
+                    {targetListLabel} {targetList}
+                </React.Fragment>
+            )}
+            </React.Fragment>
```

Comment:
Normally `<React.Fragment>` gets treated as any other HTML element, so child elements would get indented like you have below in `strandComponent`.